Tests/scripts/lock_cloud_machines.py

```diff
@@ -7,12 +7,14 @@
 import requests
 from google.cloud import storage  # noqa
 import argparse
+from Utils.github_workflow_scripts.utils import get_env_var
 
+GITLAB_SERVER_URL = get_env_var('CI_SERVER_URL', 'https://code.pan.run')  # disable-secrets-detection
```

Comment:
When the migration is over we'd need to change this default value.
(Add this to the Phase_2 issue.)